### PR TITLE
Knight: NavigationLayout

### PR DESCRIPTION
**Pull request by chineduknight**
-Added the Navbar to the ui
-Added the Sidebar to the ui
-Added the NavigationLayout to layout

[finished]

#### What does this PR achieve?
-Adds the Navbar to the ui
-Adds the Sidebar to the ui
-Adds the NavigationLayout to layout

#### Screenshots:
ftSidebar
ft02header
ft01

#### Relevant Trello Tickets
- [Create Portal Layout Component](https://trello.com/c/iSDlQKNd/99-create-portal-layout-component)

### CODE REVIEW

**Comment by chidioguejiofor** [comment]
@ChineduOguejiofor  There is a conflict in your packge.json. Please review.Also rename your Pr titile to have the format `YourName: Short Title`

File: src/components/ui/OrganizationHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import Icon from 'components/ui/Icon';
+import Logo from 'components/ui/Logo';
+import Avatar from 'components/ui/Avatar';
+
+export default function OrganizationHeader() {
+  const sampleImage =
+    'https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80';
```
Comment (chidioguejiofor) [inline src/components/ui/OrganizationHeader/index.tsx]: This should have actually been passed as a prop since we want it to change

Comment (chidioguejiofor) [reply src/components/ui/OrganizationHeader/index.tsx]: You can use this url when creating a sample image though

**Review by chidioguejiofor — COMMENTED** [review]
Looks good. Once you resolve the conflict this can be merged. Also made a single liner change.

File: src/components/ui/OrganizationHeader/index.tsx (hunk 1 above)
Comment (chineduknight) [reply src/components/ui/OrganizationHeader/index.tsx]: I have added the prop

**Review by chidioguejiofor — APPROVED** [review]

**Pull request merged**